tests/core/Observer.spec.ts

```diff
@@ -438,6 +438,56 @@ describe('Observer', async () => {
     expect(processedTransactions[3].anchorString).toEqual('4thTransaction');
   });
 
+  it('should log error if blockchian throws', async () => {
```

Comment:
Explicit test that tests the weirdly missing test.

Reply:
Sanity: Shouldn't this test be 'should rethrow if unexpected error when fetching transaction from blockchain`?

Reply:
There's a catch and log in `processTransactions`